packages/suite/src/views/dashboard/DashboardPromoBanner/TrezorExpertBanner.tsx

```diff
@@ -0,0 +1,150 @@
+import styled, { useTheme } from 'styled-components';
+
+import { resolveStaticPath } from '@suite-common/suite-utils';
+import {
+    Button,
+    Column,
+    IconButton,
+    PNG_IMAGES,
+    PNG_PATH,
```

Comment:
These should be renamed since it's not only PNGs anymore. We have there also WEBPs.